src/parser/jobs/ast/modules/Lightspeed.tsx

```diff
@@ -0,0 +1,231 @@
+		if (!lastWindow.end) {
+			lastWindow.rotation.push(event)
+			if (action.onGcd) {
+				lastWindow.gcdCount++
+			}
+			if (this.parser.patch.before('5.3') && action.mpCost != null) {
+				lastWindow.mpSavings = lastWindow.mpSavings + action.mpCost/2
+			}
+		}
+
+		// If we haven't recorded a trailing GCD event for this closed window, do so now
+		if (lastWindow.end && !lastWindow.trailingGcdEvent && action.onGcd) {
+			lastWindow.trailingGcdEvent = event
+		}
+	}
+
+	// just output, no suggestions for now.
+	override output() {
+		const tableData = this.history.map(window => {
+			const end = window.end != null ?
+				window.end - this.parser.pull.timestamp :
+				window.start - this.parser.pull.timestamp
+			const start = window.start - this.parser.pull.timestamp
+			// how long (or short, really) a window needs to be in order to be considered truncated
+			// eslint-disable-next-line @typescript-eslint/no-magic-numbers
+			const LIGHTSPEED_TRUNCATE_DURATION = this.data.statuses.LIGHTSPEED.duration - 2000
+
+			// overlapped if: we detected an overwrite of this player onto another player, or if
+			// this player's buff had a duration that was too short and they didn't die
+			const overlap =  (!window.deathTruncated && (end !== start) && (end - start < LIGHTSPEED_TRUNCATE_DURATION))
+
+			return ({
+				start,
+				end,
+				overlap,
+				notesMap: {
+					overlapped: <>{overlap ? <Icon name="x" color="red" /> : <Icon name="check" color="green" />}</>,
+					totalSavings: <>{window.mpSavings}</>,
+				},
+				rotation: window.rotation,
+				targetsData: {
+					gcds: {
+						actual: window.gcdCount,
+						expected: (BASE_GCDS_PER_WINDOW),
+					},
+				},
+			})
+		})
+
+		const actualCasts = tableData.length
+		const totalPossibleCasts = Math.ceil(this.parser.pull.duration / this.data.actions.LIGHTSPEED.cooldown)
+
+		const noCastsMessage = <p><span className="text-error"><Trans id="ast.lightspeed.messages.no-casts"> There were no casts recorded for <DataLink action="LIGHTSPEED" />.</Trans></span></p>
+		const castsMessage = <p><Trans id="ast.lightspeed.messages.num-casts"> There were a total of {actualCasts} out of a possible {totalPossibleCasts} <DataLink action="LIGHTSPEED" /> casts noted.</Trans></p>
+
+		const pre5_3message = <p><Trans id="ast.lightspeed.messages.explanation">
+		The main use of <DataLink action="LIGHTSPEED" /> should be for weaving card actions during <DataLink action="DIVINATION" /> and <DataLink action="SLEEVE_DRAW" /> windows.<br />
+		It can also be used for MP savings on heavy healing segments, keeping casts up while on the move and other specific scenarios.<br />
+		Each fight calls for a different strategy, but try to utilize it as much as possible.<br />
+		Unless it's being used for <DataLink action="ASCEND" />, lightspeed should fit at least 6 GCDs.<br />
+		</Trans></p>
+
+		const message = <p><Trans id="ast.lightspeed.messages.explanation.ost5_3">
```

Comment:
```suggestion
		const message = <p><Trans id="ast.lightspeed.messages.explanation.post5_3">
```

Unless this actually is an original soundtrack, in that case carry on. It's a bit weird that we invert the localisation key from the variable name tho, consider renaming these so message gets explanation and pre5_3message gets pre5_3explanation or something. You'll likely need to edit these in the locale files manually.

Reply:
updated both this and the relating locale files
yarn extract was ran after doing that, but probably good to review quickly just in case